wikipedia-dark.user.css

```diff
@@ -202,11 +202,14 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
     background-color: transparent !important;
     color: var(--gray-9a);
   }
+  .mw-mmv-info-box {
+    border-color: var(--gray-5) !important;
+  }
   .mw-mmv-image-metadata, .navbox, .navbox-subgroup, .mw-mmv-info-box,
   .mwe-pt-article-row-odd, .mwe-pt-article-row-even, .ve-ui-mwSaveDialog-options,
   #mw-page-base, .mw-parser-output #siteContainer div:first-child,
   div.vectorTabs li.selected {
-    background-color: transparent;
+    background-color: transparent !important;
```

Comment:
Please dont add !important to any rules like this, the correct way is to isolate the desired selector and increase its specificity and only by last resort use !important.

Reply:
I've isolated the selector.